src/mappers/ideology/ideology_mapper.cpp

```diff
@@ -51,4 +51,27 @@ std::string mappers::IdeologyMapper::GetRulingIdeology(const std::set<std::strin
    });
 
    return winning_ideology->first;
+}
+
+
+std::string mappers::IdeologyMapper::GetSubIdeology(const std::string& ideology,
+    const std::set<std::string>& current_laws) const
+{
+   const auto rules = sub_ideology_rules_.find(ideology);
+   if (rules == sub_ideology_rules_.end())
+   {
+      return "despotism";
```

Comment:
You could, technically, have multiple fallback values for different mainline ideologies. Despotism for nonaligned only.